components/dashboard/src/Analytics.tsx

```diff
@@ -35,7 +35,7 @@ export const getAnonymousId = (): string => {
   }
   else {
     anonymousId = v4();
-    Cookies.set('ajs_anonymous_id', anonymousId);
+    Cookies.set('ajs_anonymous_id', anonymousId, {domain: '.'+window.location.hostname, expires: 365});
```

Comment:
Side note: Wouldn't `path: ''` also be needed? Or is it okay to always set a different path depending on what the current page is?